packages/color/tsconfig.json

```diff
@@ -0,0 +1,10 @@
+{
+  "compilerOptions": {
+    "strict": true,
+
+    "resolveJsonModule": true,
+    "esModuleInterop": true,
+    "moduleResolution": "node"
+  },
+  "include": ["src/**/*.ts", "src/**/*.tsx", "src/**/*.d.ts"]
```

Comment:
Did you run into issues without that explicit `include` ?

Reply:
Yup, tests get built.
```
dist
  src
  test
    index.test.d.ts
```

Reply:
Makes sense! Might want to add that to other files in a follow-up PR then

Reply:
Since this is a monorepo, we could add `tsconfig.json` to the root with all _base_ compiler options (like `moduleResolution node` and no `resolveJsonModule`) and then use [`"extends"`](https://www.typescriptlang.org/docs/handbook/tsconfig-json.html#configuration-inheritance-with-extends) to inherit them. The tsconfig in root would be useful when you open the entire repo in an IDE.

I'm not sure for 100%, but I think "include" is inherited literally (relative to the new config, not to the parent), so most tsconfigs in packages could look like
```
{ 
  "extends": "../core/tsconfig.json",
  "compilerOptions": {
    "strict": true // if possible since core is unstrict
  }
}
```

Reply:
Yes we should totally do this! 👍